src/metabase/cmd/dump_to_h2.clj

```diff
@@ -34,6 +36,10 @@
      (when-not keep-existing?
        (copy.h2/delete-existing-h2-database-files! h2-filename))
      (copy/copy!  (mdb.conn/db-type) (mdb.conn/jdbc-spec) :h2 h2-jdbc-spec)
-     (when dump-plaintext?
-       (copy/overwrite-encrypted-fields-to-plaintext! (mdb.conn/db-type) (mdb.conn/jdbc-spec) :h2 h2-jdbc-spec))
-     (println "Dump complete"))))
+     (if dump-plaintext?
+       (binding [mdb.conn/*db-type* :h2
+                 mdb.conn/*jdbc-spec* h2-jdbc-spec
+                 db/*db-connection* h2-jdbc-spec
+                 db/*quoting-style* :h2]
+         (when (not (rotate-encryption/rotate-encryption-key! nil))
+           1))))))
```

Comment:
This is kind of weird to return a number like this. Is this supposed to be the exist code? Like Rob suggested, you should just throw exceptions if stuff goes wrong and then have the top level of the command catch them and then print them and exit with a nonzero status

Reply:
aha, yes, it felt strange when I was doing it already. I'll change it to exception being thrown.